samples/bluetooth/bthome_sensor_template/src/main.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/uuid.h>
+
+#define SERVICE_DATA_LEN        9
+#define SERVICE_UUID            0xfcd2		/* BTHome service UUID */
```

Comment:
This could probably go into `bluetooth/uuid.h` since it's a 16-bit assigned UUID

Reply:
Yes, I can do that but I thought uuid.h only contained UUID's from the BT spec.

Reply:
I agree. BTHome uses a a 16-bit UUID sponsored by Allterco. It should not go into `bluetooth/uuid.h`.

Reply:
my bad, I wasn't sure.